hoomd/ClockSource.h

```diff
@@ -24,16 +24,87 @@
 
 #include <iostream>
 
+#ifdef __GNUC__
 #include <sys/time.h>
 #include <unistd.h>
+#endif
+#ifdef _WIN32
+#include <Windows.h>
+#include <time.h>
+#include <io.h>
+#endif
 
 #include <hoomd/extern/pybind/include/pybind11/pybind11.h>
 
+
+#ifdef _WIN32
+inline void usleep(__int64 usec)
```

Comment:
A cleaner solution would be to re-implement the ClockSource.h methods using the cross-platform C++11 chrono (https://en.cppreference.com/w/cpp/chrono)